Scripts/Insights_19.10.3_hotfix.ps1

```diff
@@ -0,0 +1,52 @@
+﻿param(
+   [Parameter(Mandatory=$true)][string]$username
+  ,[Parameter(Mandatory=$true)][string]$password
+)
+
+function Get-checkBuild($tenant)
+{
+        $cubeBuilding = 0
+        while ($cubeBuilding -ne 1)
+        {
+            $buildResult = & "C:\Program Files (x86)\UiPath\Orchestrator\Tools\UiPath.InsightsAdminTool.exe" buildStatus -t $tenant | out-string
```

Comment:
Better to use a system variable to get to Program Files folder

Reply:
I think we need to, otherwise i will not work